docs/conf.py

```diff
@@ -0,0 +1,60 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+project = "pynntp"
+version = "2.0.1"
+copyright = "2024, Byron Platt"  # noqa: A001
+author = "Byron Platt and contributors"
```

Comment:
By using `sphinx-pyproject` https://pypi.org/project/sphinx-pyproject/
We only need to have the version in one place, and basically all of this config can live in `pyproject.toml`

`docs/conf.py` would become

```python
from sphinx_pyproject import SphinxConfig

config = SphinxConfig("../pyproject.toml", style="poetry", globalns=globals())
project = config.name
release = config.version
```

and the rest of the config moves to pyproject.toml `[tool.sphinx-pyproject]` section